atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -73,11 +73,13 @@ def run(self):
 
                 digests = get_manifest_digests(registry_image, registry,
                                                insecure, docker_push_secret)
-
                 tag = registry_image.to_str(registry=False)
                 push_conf_registry.digests[tag] = digests
 
+            first_registry_image = self.workflow.tag_conf.images[0]
+            push_conf_registry.config = get_config_from_registry(
+                first_registry_image, registry, digests.v2, insecure,
```

Comment:
No, v2 schema 1 is no good. The config object is not available that way.

If the schema 2 manifest is not available, no `config` key should be added to the extra metadata.